### PR TITLE
Add glcontext 2.2.0

### PR DESCRIPTION
**Pull request by patricksnape**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by patricksnape** [comment]
This is required to unblock https://github.com/conda-forge/moderngl-feedstock/issues/11

**Comment by patricksnape** [comment]
@conda-forge/help-python Ready for review

**Comment by patricksnape** [comment]
@conda-forge/help-python Re-ping that this is ready for review :)

**Comment by patricksnape** [comment]
@ocefpaf Maybe you can help me? :D

File: recipes/glcontext/meta.yaml
Comment (isuruf) [inline recipes/glcontext/meta.yaml:44]: @cprogrammer1994, are you okay with being a maintainer?

**Review by isuruf — APPROVED** [review]
Looks good.

**Comment by ocefpaf** [comment]
> @ocefpaf Maybe you can help me? :D

Sorry, still catching up with my notifications. Nice to see you around here again BTW.
And thanks @isuruf for the review!

**Comment by patricksnape** [comment]
Thanks @isuruf and @ocefpaf !

**Pull request merged**
